### DISCUSSION TITLE
Do the new country flag changes apply to osu!lazer?

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#25593] [Q&A] [answered]
https://osu.ppy.sh/home/news/2023-07-28-enabling-country-flag-changes

> if you've been consistently connecting from a country that differs from the country associated with your account over a long period of time (currently this is 6 months), you will be prompted to change your country flag in the account settings page

1. Does this apply to osu!lazer? If I open osu!lazer every once in a while for 6 months, will I be able to change my country flag? Or does this only apply to osu!stable?

2. Is it also sufficient to just open some https://osu.ppy.sh/ page while being logged in every once in a while for 6 months? Or do I need to open the game?

3. How often do I need to connect over the period of 6 months?

**Comment by peppy** [comment, marked as answer]
Sorry but the specific conditionals or this are intentionally not revealed. If you continue to play from a country the option will become available to you.
